add-on/chrome/bootstrap.js

```diff
@@ -616,7 +616,7 @@ var WindowListener = {
         this.activeSound.load();
         this.activeSound.play();
 
-        this.activeSound.addEventListener("ended", () => this.activeSound = undefined, false);
+        this.activeSound.addEventListener("ended", () => { this.activeSound = undefined; }, false);
```

Comment:
Do you know what change resulted in needing these fixed? `"no-return-assign": 2,` was already active in `.eslintrc`... ?